pkg/lisp/go.mod

```diff
@@ -0,0 +1,20 @@
+module github.com/lf-edge/eve/pkg/lisp
+
+go 1.12
+
+require (
+	github.com/google/gopacket v1.1.16
+	github.com/lf-edge/eve/pkg/pillar v0.0.0-20200203060116-d5cb7c3fb58a
+	github.com/sirupsen/logrus v1.4.2
+	golang.org/x/net v0.0.0-20191004110552-13f9640d40b9
+)
+
+replace github.com/lf-edge/eve/api/go => ../../api/go
+
+// replace github.com/lf-edge/eve/pkg/pillar => ../../../../../pkg/pillar
```

Comment:
Why does this depend on pillar? Is it some of the libs we use? And could those be moved entirely to lisp, or are they shared, e.g. pubsub?